sklearn/tree/export.py

```diff
@@ -60,7 +61,13 @@ def _color_brew(n):
     return color_list
 
 
-def export_graphviz(decision_tree, out_file="tree.dot", max_depth=None,
+class Sentinel:
```

Comment:
For python 2.7, this yields an old class: let's stick to `class Sentinel(object)` for now, as it will not yield exactly the same class in python 2 and python 3 with that syntax.